src/noframe.js

```diff
@@ -12,22 +12,27 @@ export default function noframe(target, container) {
   if (!('length' in frames)) frames = [frames];
   for (let i = 0; i < frames.length; i += 1) {
     const frame = frames[i];
-    let parent = frame.parentElement;
     const h = frame.offsetHeight;
     const w = frame.offsetWidth;
     const styles = frame.style;
     let maxW = `${w}px`;
+    let parent = null;
+    // If a targeted container <element> is present in DOM
     if (typeof container !== 'undefined') {
```

Comment:
If the `container` argument is optional and the `parent` **will** have a value—there **should not be** statement logic based around the `parent` **not** having a falsy value.

Reply:
@yowainwright Ok let me change code as you suggested.